gcc/rust/typecheck/rust-tyty-subst.cc

```diff
@@ -588,6 +588,16 @@ SubstitutionRef::get_used_arguments () const
 {
   return used_arguments;
 }
+tl::optional<SubstitutionArg>
```

Comment:
can you add a space here between the previous block and this one

Reply:
Fixed